src/legacy/core_plugins/kibana/public/discover/doc_table/components/pager/tool_bar_pager_text.tsx

```diff
@@ -16,5 +16,23 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
 
-import './tool_bar_pager_buttons';
+interface Props {
+  startItem: number;
+  endItem: number;
+  totalItems: number;
+}
+
+export function ToolBarPagerText({ startItem, endItem, totalItems }: Props) {
+  return (
+    <div className="kuiToolBarText" data-test-subj="toolBarPagerText">
```

Comment:
I think we should be able to use EuiText to format this text.

Reply:
Thx, I will address this in the EUIFication PR